packages/apps/dev-wallet/src/pages/transaction/components/Signer.tsx

```diff
@@ -81,7 +81,7 @@ export const RenderSigner = ({
         {signer.clist &&
           signer.clist.map((cap) => (
             <Stack
-              key={cap.name}
+              key={`${cap.name}${cap.args.toString()}`}
```

Comment:
don't think this does what you intend for it to do, I think you'd rather want `cap.args.join(',')` as this result in `[object Object]`, right?